mod_dynamic_spawns/classes/unit_block.nut

```diff
@@ -114,14 +114,23 @@
 		}
 		else
 		{
-			local function satisfiesTierWidth( _index )
+			local spawnables = this.__DynamicSpawnables.filter(@(_, _unit) _unit.canSpawn());
+			if (spawnables.len() == 0)
+				return null;
+
+			local function satisfiesTierWidth( _idx )
```

Comment:
this function needs documentation on what it exactly wants to do
I dont understand yet what this wants to do

Reply:
I've added a comment explaining the function's logic.